spec/index.html

```diff
@@ -3674,7 +3674,8 @@ <h3>ORDER BY</h3>
         <p>The <a href="#op_lt">"&lt;" operator</a> (see the <a href="#OperatorMapping">Operator
             Mapping</a> and <a href="#operatorExtensibility">17.3.1 Operator Extensibility</a>) defines
           the relative order of pairs of <code>numerics</code>,
-          <code>xsd:strings</code>, <code>xsd:booleans</code> and <code>xsd:dateTimes</code>. Pairs of
+          <code>xsd:strings</code>, <code>xsd:booleans</code>, <code>xsd:dateTimes</code>, and
+          <a data-cite="RDF12-CONCEPTS#dfn-triple-term">Triple Terms</a>. Pairs of
           IRIs are ordered by comparing them as literals with datatype <code>xsd:string</code>.</p>
```

Comment:
I think we need to say something about case-_sensitive_ ordering being the norm in some environments, and case-_insensitive_ ordering in others, as well as how to override both norms, assuming that is possible anywhere if not everywhere. I know we've said in some place(s) that lexical ordering is to be code point order, which is effectively case-sensitive, but I don't think this will suffice for all use cases. So far as I knowI believe code point order will also prove problematic when trying to compare, for instance, `ä`, as a single character and as a composite character (i.e., a combination of `a` and `¨`), and in making *all the `a`s (uppercase, lowercase, with diacriticals, etc.) appear as a group and be in a predictable order within that group*.